### PR TITLE
Decouple create budget call from enable()

### PR DESCRIPTION
**Pull request by junjizhi**
Addresses https://github.com/statechannels/monorepo/issues/1453

### CODE REVIEW

File: packages/web3torrent/src/pages/file/File.tsx [hunk 1]
```diff
@@ -52,6 +52,13 @@ const File: React.FC<Props> = props => {
     })
   );
 
+  useEffect(() => {
+    const initResult = async () => {
+      await web3Torrent.init();
+    };
+    initResult();
+  }, [web3Torrent]);
```
Comment (junjizhi) [inline packages/web3torrent/src/pages/file/File.tsx:60]: Not sure if this is the correct usage of useEffect, but I will leave it for now.

**Comment by lalexgap** [comment]
@junjizhi  just a heads up that I've done something similar in my PR https://github.com/statechannels/monorepo/pull/1528

File: packages/web3torrent/src/pages/file/File.tsx (hunk 1 above)
Comment (snario) [reply packages/web3torrent/src/pages/file/File.tsx:60]: cc @kerzhner / @geoknee / @spersico

Comment (spersico) [reply packages/web3torrent/src/pages/file/File.tsx:60]: I think that Alex's PR supersedes this one. Might be useful to coordinate efforts to merge both pieces of code

Comment (junjizhi) [reply packages/web3torrent/src/pages/file/File.tsx:60]: Good catch. This is mostly an investigation so the code here is not polished. I will focus on Alex's PR instead.

**Pull request closed without merging**